ecs.php

```diff
@@ -44,43 +46,45 @@
         __DIR__ . '/ecs.php',
     ]);
 
-    $parameters->set(Option::EXCLUDE_PATHS, [
+    $containerConfigurator->import(SetList::COMMON);
+    $containerConfigurator->import(SetList::CLEAN_CODE);
+    $containerConfigurator->import(SetList::PSR_12);
+    $containerConfigurator->import(SetList::ARRAY);
+
+    $parameters->set(Option::SKIP, [
         'packages/*/var/*php',
         '*/vendor/*.php',
         __DIR__ . '/packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php',
-    ]);
 
-    $parameters->set(Option::SETS, [
-        SetList::COMMON,
-        SetList::CLEAN_CODE,
-        SetList::PHP_70,
-        SetList::PHP_71,
-        SetList::PSR_12,
-        SetList::DEAD_CODE,
-        SetList::ARRAY,
-    ]);
-
-    $parameters->set(Option::SKIP, [
-        NotOperatorWithSuccessorSpaceFixer::class => null,
-        CastSpacesFixer::class => null,
-        OrderedClassElementsFixer::class => null,
-        NoSuperfluousPhpdocTagsFixer::class => null,
-        PhpdocVarWithoutNameFixer::class => null,
-        PhpUnitStrictFixer::class => null,
-        BlankLineAfterOpeningTagFixer::class => null,
-
-        MethodChainingIndentationFixer::class => ['*/Configuration.php'],
-
-        MethodChainingNewlineFixer::class => [
-            // bug, to be fixed in symplify
-            '*/Configuration.php',
-            __DIR__ . '/packages/EasyCore/tests/Doctrine/DBAL/Types/DateTimeMicrosecondsTypeTest.php',
+        NotOperatorWithSuccessorSpaceFixer::class,
+        CastSpacesFixer::class,
+        OrderedClassElementsFixer::class,
+        NoSuperfluousPhpdocTagsFixer::class,
+        PhpdocVarWithoutNameFixer::class,
+        PhpUnitStrictFixer::class,
+        BlankLineAfterOpeningTagFixer::class,
+        MethodChainingNewlineFixer::class,
+        ClassAttributesSeparationFixer::class,
+        ArrayOpenerAndCloserNewlineFixer::class,
```

Comment:
This is the replacement for `ArrayOpenerNewlineFixer`. But it converts all the single-line arrays into muti-line ones:
```
-                ->addTag('monolog.logger', ['channel' => BridgeConstantsInterface::LOG_CHANNEL]);
+                ->addTag('monolog.logger', [
+                    'channel' => BridgeConstantsInterface::LOG_CHANNEL,
+                ]);
```